app/src/components/user-page/change-user-password-modal.vue

```diff
@@ -119,7 +122,8 @@ export default {
         async changePassword() {
             try {
                 const newUser = this.user;
```

Comment:
```suggestion
                const newUser = Object.assign({}, this.user);
```

newUser should be a new object, so we don't mutate this.user.